action.yml

```
@@ -56,64 +72,86 @@ runs:
         filter=${{ inputs.filter }}
         labels=${{ inputs.labels }}
         includeUntagged=${{ inputs.includeUntagged }}
+        sinceLatestRelease=${{ inputs.sinceLatestRelease }}
+        verbose=${{ inputs.verbose }}
+        branch=${{ inputs.branch }}
+        excludedLabels=${{ inputs.excludedLabels }}
+
+        # add space escapes to labels comma separated array
```

Did you mean to have something like this?

```suggestion
        # add space escapes the labels comma separated array
```

---

No, this changes string "Bug fix,other fix" into "Bug\ fix,other\ fix". So it adds space escape to the string

---

I see, thanks for clarifying